addons/website_slides/controllers/thread.py

```diff
@@ -0,0 +1,37 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.addons.mail.controllers import thread
+from odoo import _, http
+from odoo.http import request
+from odoo.exceptions import ValidationError
+
+
+class ThreadController(thread.ThreadController):
+
+    @http.route()
+    def mail_message_post(self, thread_model, thread_id, post_data, context=None, **kwargs):
+        previous_post = request.env["mail.message"].search(
+            [
+                ("res_id", "=", thread_id),
+                ("author_id", "=", request.env.user.partner_id.id),
+                ("model", "=", "slide.channel"),
+                ("subtype_id", "=", request.env.ref("mail.mt_comment").id),
+            ]
+        )
+        if thread_model == "slide.channel" and previous_post:
```

Comment:
When I see all of those conditions based on thread model, it feels like this whole override should be done as an override of `message_post` in slide model instead.

Reply:
As discussed, this function uses the `pid` which is not accessible in `message_post` so we let it be here for now.